src/Main.elm

```diff
@@ -143,11 +135,15 @@ saveAction : Model -> Action -> Model
 saveAction model action =
     case model.actionA of
         Nothing ->
-            { model
-                | valueA = displatToFloat model.display
-                , actionA = Just action
-                , displayStatus = ShowingResults
-            }
+            if action == Equals then
+                model
```

Comment:
This is a situation where `case` is more fluent because it works directly on the data you have, without you needing to use `==` to manufacture a boolean. In contrast, in the earlier situation we discussed you were always going to have a `Boolean` in hand, so you might as well use `if` on it.